### DISCUSSION TITLE
[A-L] (2024/25) Foglio 2 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Rurik-D** [sapienzastudentsnetwork/algebra#480] [Esercizi - Viaggi/Cherubini]
![image](https://github.com/user-attachments/assets/fde680a2-b5e8-47bd-a988-a46259b53e68)

**Comment by matypist** [comment]
Soluzione ad opera della [tutor](https://github.com/sapienzastudentsnetwork/algebra/blob/main/scandiuzzi/README.md#aa-202425-scandiuzzi-tutor) Elisa **Scandiuzzi**, tutoraggio del **9 ottobre** 2024 ([appunti_24_10_09.pdf](https://drive.google.com/file/d/1tqiXXi2WFl-N2J1HJcQ47xS_EOnubLaC/view?usp=drive_link)):

![image](https://github.com/user-attachments/assets/e875b187-c0c2-42ed-90f7-f37125b708b3)
![image](https://github.com/user-attachments/assets/d47c9a78-7099-439d-b820-4bd1c739e344)
